src/main/java/uk/gov/laa/ccms/caab/service/LookupService.java

```diff
@@ -258,7 +256,172 @@ public Mono<CommonLookupDetail> getDisabilities() {
    * @return CommonLookupValueDetail containing the common lookup value.
    */
   public Mono<CommonLookupValueDetail> getDisability(String code) {
-    return ebsApiClient.getCommonValues(COMMON_VALUE_DISABILITY, code)
+    return getCommonValue(COMMON_VALUE_DISABILITY, code);
+  }
+
+
+
+  /**
+   * Get a list of Levels Of Service Common Values.
+   *
+   * @return CommonLookupDetail containing the common lookup values.
+   */
+  public Mono<CommonLookupDetail> getLevelsOfService() {
+    return ebsApiClient.getCommonValues(COMMON_VALUE_LEVEL_OF_SERVICE);
+  }
+
+  /**
+   * Get a list of Matter Type Common Values.
+   *
+   * @return CommonLookupDetail containing the common lookup values.
+   */
+  public Mono<CommonLookupDetail> getMatterTypes() {
+    return ebsApiClient.getCommonValues(COMMON_VALUE_MATTER_TYPES);
+  }
+
+  /**
+   * Get a list of Client Involvement Types Common Values.
+   *
+   * @return CommonLookupDetail containing the common lookup values.
+   */
+  public Mono<CommonLookupDetail> getClientInvolvementTypes() {
+    return ebsApiClient.getCommonValues(COMMON_VALUE_CLIENT_INVOLVEMENT_TYPES);
+  }
+
+  /**
+   * Get a list of Scope Limitation Common Values.
+   *
+   * @return CommonLookupDetail containing the common lookup values.
+   */
+  public Mono<CommonLookupDetail> getScopeLimitations() {
+    return ebsApiClient.getCommonValues(COMMON_VALUE_SCOPE_LIMITATIONS);
+  }
+
+  /**
+   * Get a list of Scope Limitation Detail.
+   *
+   * @return ScopeLimitationDetails containing the scope limitation details.
+   */
+  public Mono<ScopeLimitationDetails> getScopeLimitationDetails(
+      ScopeLimitationDetail searchCriteria) {
+    return ebsApiClient.getScopeLimitations(searchCriteria);
+  }
+
+  /**
+   * Get a list of Proceeding Status Lookup Values.
+   *
+   * @return CommonLookupDetail containing the common lookup values.
+   */
+  public Mono<CommonLookupDetail> getProceedingStatuses() {
+    return ebsApiClient.getCommonValues(COMMON_VALUE_PROCEEDING_STATUS);
+  }
+
+  /**
+   * Get a single Proceeding Status Lookup Value by code.
+   *
+   * @return CommonLookupValueDetail containing the status value.
+   */
+  public Mono<CommonLookupValueDetail> getProceedingStatus(String code) {
```

Comment:
add param to java doc

add finals to params

Reply:
done